apple/xcode/osx/Outline.xcodeproj/project.pbxproj

```diff
@@ -688,6 +688,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				EXCLUDED_ARCHS = arm64;
```

Comment:
Why do we need to exclude arm64?
Could you add a comment here and in the other place?

Reply:
arm64 is the new Apple Sillicon architecture. Our framework dependencies (third and first party) don't support it yet. Added the comments, though they won't show up on Xcode.